docs/integrations/destinations/azure-blob-storage.md

```diff
@@ -151,6 +151,7 @@ With the field `File Extension`, it is possible to save the output files with ex
   <summary>Expand to review</summary>
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                                         |
+| 0.2.6 | 2025-03-24 | [56355](https://github.com/airbytehq/airbyte/pull/56355) | Upgrade to airbyte/java-connector-base:2.0.1 to be M4 compatible. |
```

Comment:
This formatting is wrong. Please move the new changelog line below the line break and formatting on line 155. Do this for all the .md files. Feel free to commit directly to this branch.